### DISCUSSION TITLE
scaling based on http request available in container apps

### DISCUSSION THREAD

**Discussion by dariuszbz** [kedacore/http-add-on#388] [General] [answered]
![image](https://user-images.githubusercontent.com/19564162/151774847-b9f79294-6464-4b27-bedc-e8e7d46ad2e3.png)

**Comment by tomkerkhove** [comment]
Can you elaborate on your question please?

**Comment by dariuszbz** [comment, marked as answer]
Lol. I thought it was discussion board. Sorry for messing around!
